src/test/perl/modules/filetools.pm

```diff
@@ -0,0 +1,33 @@
+package filetools;
```

Comment:
@jrha @ned21 hmm, i can't request changes on my own PR. can you help me not to forget 😄